src/rules/Bytes32StringRules.ts

```diff
@@ -0,0 +1,12 @@
+import { utils } from 'ethers';
+import ApiError from '../exceptions/ApiError';
+
+export function validateBytes32StringRule(string: string) {
```

Comment:
We should avoid variable names that are the same as a type. How about `str: string` ?